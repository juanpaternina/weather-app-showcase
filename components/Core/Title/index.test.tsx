import { render, screen } from '@testing-library/react-native';

const { Title } = require('.');

describe('Title', () => {
  test('It should render a Text component', () => {
    render(<Title>Title</Title>);
    const text = screen.getByText('Title');
    expect(text).toBeDefined();
  });
});
