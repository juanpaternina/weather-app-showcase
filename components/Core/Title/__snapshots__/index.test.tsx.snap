// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Title It should render a Text component 1`] = `
<Text
  style={
    {
      "color": "black",
      "fontFamily": "bold",
      "fontSize": 24,
      "fontWeight": "bold",
    }
  }
>
  Title
</Text>
`;
